### PR TITLE
[FIX] logs in pos ms

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_multi_session/static/src/js/pos_multi_session.js [hunk 1]
```diff
@@ -111,23 +115,24 @@ odoo.define('pos_multi_session', function(require){
...
                     });
                 }
                 if (sync_all) {
                     this.message_ID = data.message_ID;
                     this.ms_do_update(order, data);
                 } else {
-                    if (self.message_ID + 1 != data.message_ID)
-                        self.multi_session.request_sync_all();
-                    else
+                    if (self.message_ID + 1 === data.message_ID){
```
Comment (yelizariev) [inline pos_multi_session/static/src/js/pos_multi_session.js:126]: You probably wanted to use ``!==``, because otherwise you change code

Comment (yelizariev) [reply pos_multi_session/static/src/js/pos_multi_session.js:126]: I got it. You replaced blocks. Please don't do that. Use ``!==`` instead

Comment (KolushovAlexandr) [reply pos_multi_session/static/src/js/pos_multi_session.js:126]: There was that lint error: Unexpected negated condition. [Error/no-negated-condition], so I fixed it according to this http://eslint.org/docs/2.0.0/rules/no-negated-condition.
Ok, I won't

Comment (KolushovAlexandr) [reply pos_multi_session/static/src/js/pos_multi_session.js:126]: @yelizariev What about logs? are they ok?
That issue https://github.com/it-projects-llc/pos-addons/issues/327   https://www.it-projects.info/web#id=1645&view_type=form&model=project.task also was fixed with lints.

Comment (yelizariev) [reply pos_multi_session/static/src/js/pos_multi_session.js:126]: logs are fine.

Ok, I didn't know about no-negated-condition.

File: pos_multi_session/static/src/js/pos_multi_session.js [hunk 2]
```diff
@@ -451,10 +470,10 @@ odoo.define('pos_multi_session', function(require){
                     }
                     self.request_sync_all();
                 } else {
-                    if (!self.offline_sync_all_timer) {
-                        var warning_message = _t("No connection to the server. You can create new orders only. It is forbidden to modify existing orders.");
-                        self.warning(warning_message);
-                        self.start_offline_sync_timer();
+                    self.no_connection_warning();
+                    self.start_offline_sync_timer();
```
Comment (yelizariev) [inline pos_multi_session/static/src/js/pos_multi_session.js]: Have you removed test ``if (!self.offline_sync_all_timer) {``? Why?

**Comment by yelizariev** [comment]
WIP

**Pull request closed without merging**